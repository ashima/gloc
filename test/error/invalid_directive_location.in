void main() { #error directives must be first semantic token on line
       #version 100 // so this is fine
