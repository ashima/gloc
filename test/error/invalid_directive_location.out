File 0, line 1, col 14:
invalid directive location

